html-css/override-extend-variables/step4-instructions.md

```diff
@@ -1,11 +1,11 @@
 ## Add a PatternFly class for the layout
-
-Now that we have an updated Card design, let's add several Cards and use the gallery layout to organize them on the page. 
-
+ 
+Now that you have an updated Card design, let's add several Cards and use the gallery layout to organize them on the page. 
+ 
 1) <strong>Copy code into the index.html file.</strong>
-
-Click the <strong>Copy to Editor</strong> button below to add html for multiple Cards to the index.html file.
-
+ 
+2) <strong>Click the Copy to Editor button </strong> below to add html for multiple Cards to the index.html file.
```

Comment:
bold entire sentence

Reply:
<strong>Click the Copy to Editor button </strong> below to add html for multiple Cards to the index.html file.
sb
<strong>Click the Copy to Editor button  below to add html for multiple Cards to the index.html file.</strong>